content-src/components/DiscoveryStreamComponents/SafeAnchor/SafeAnchor.jsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+
+export class SafeAnchor extends React.PureComponent {
+  safeURI(url) {
+    try {
+      const {protocol} = new URL(url);
```

Comment:
nit: only wrap the call the `new URL` in a try catch. imo it's best to keep the try catch block as small as possible so that it doesn't accidentally catch things we are not expecting.

Reply:
@piatra done, thanks!